cmd/k8sagent/init/command.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2012-2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package init
```

Comment:
Oh... while this may work it gives me the heebies. Given that "init" is a reserved word for functions.

Perhaps naming the package "initialize"?